tests/test_shipments.py

```diff
@@ -38,6 +40,23 @@
 google_url = f'https://maps.googleapis.com/maps/api/geocode/json'
 
 
+private_key = b"""-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Please triple check that this private key isn't used anywhere else for anything, ever.